planet/cli/data.py

```diff
@@ -295,6 +296,29 @@ async def search_create(ctx, name, item_types, filter, daily_email, pretty):
         echo_json(items, pretty)
 
 
+@data.command()
+@click.pass_context
+@translate_exceptions
+@coro
+@click.argument("item_types", type=types.CommaSeparatedString())
+@click.argument('interval', type=click.Choice(STATS_INTERVAL))
+@click.argument("filter", type=types.JSON(), default="-", required=False)
+async def stats(ctx, item_types, interval, filter):
+    """Get a bucketed histogram of items matching the filter.
+
+    This function returns a bucketed histogram of results based on the
+    item_types, interval, and json filter specified (using file or stdin).
+    This function outputs a full JSON description of the returned statistics
+    result.
```

Comment:
This seems to read like two different descriptions for the function response (aka "returns" and "outputs"). Maybe delete one of them?